src/openforms/submissions/tests/form_logic/test_binding.py

```diff
@@ -0,0 +1,375 @@
+from functools import lru_cache
+from pathlib import Path
+from typing import Any
+from unittest import skip
+from urllib.parse import unquote
+
+from django.test import SimpleTestCase
+
+import requests_mock
+from factory.django import FileField
+from furl import furl
+from hypothesis import example, given, settings, strategies as st
+from zgw_consumers.constants import APITypes, AuthTypes
+
+from openforms.forms.tests.factories import FormVariableFactory
+from openforms.registrations.contrib.zgw_apis.tests.factories import ServiceFactory
+from openforms.variables.constants import DataMappingTypes
+from openforms.variables.tests.factories import ServiceFetchConfigurationFactory
+from openforms.variables.validators import HeaderValidator, ValidationError
+
+from ...form_logic import bind
+
+DEFAULT_REQUEST_HEADERS = {
+    "Accept",
+    "Accept-Encoding",
+    "Connection",
+    "Content-Type",
+    "User-Agent",
+}
+
+
+def data_mapping_values() -> st.SearchStrategy[Any]:
+    "Generates values for the value side of openforms.typing.DataMapping"
+    return st.one_of(st.text(), st.integers(), st.floats(), st.dates(), st.datetimes())
+
+
+# the first test still takes too much time
+# decorating given because I don't want to count on test running order
+given = settings(deadline=500)(given)
+
+
+class ServiceFetchConfigVariableBindingTests(SimpleTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.service = ServiceFactory.build(
```

Comment:
I get nervous when the super classmethods are not called... possibly related to your tearDownClass problem?